src/components/shared/avatar-upload.tsx

```
@@ -0,0 +1,95 @@
+"use client";
+
+import { useState } from "react";
+import { Camera, CloudUpload, Loader2 } from "lucide-react";
```

ESLint

[error] 'Camera' is defined but never used.

(@typescript-eslint/no-unused-vars)